redfishpower/redfishpower.c

```diff
@@ -622,6 +630,23 @@ static void settimeout(char **av)
   }
 }
 
+static void setwaituntildelay(char **av)
+{
+  if (av[0]) {
+      char *endptr;
+      long tmp;
+
+      errno = 0;
+      tmp = strtol (av[0], &endptr, 10);
+      if (errno
+          || endptr[0] != '\0'
+          || tmp <= 0
+          || (tmp > (LONG_MAX / USEC_IN_SEC)))
+        printf("invalid wait until delay specified\n");
+      wait_until_delay = tmp * USEC_IN_SEC;
+  }
+}
```

Comment:
This probably should be called a "status polling interval" or something more intuitive?